ReSharper.FSharp/src/FSharp.Psi.Features/src/Util/PsiUtil.fs

```diff
@@ -200,6 +200,13 @@ let rec getFirstMatchingNodeBefore (predicate: ITreeNode -> bool) (node: ITreeNo
     else
         node
 
+let getAllMatchingNodesAfter predicate (node: ITreeNode) =
```

Comment:
Wouldn't it be better to find the last node and use `TreeRange(first, last)` here?